website/templates/util/render_node.mako

```diff
@@ -143,13 +143,13 @@
         class="project list-group-item list-group-item-node">
         <p class="list-group-item-heading f-w-lg">
             %if summary['is_registration']:
-                Private Registration
+                Unshared Registration
```

Comment:
I remember a brief discussion about this. I think the problem was that
private has a very specific meaning in the context of the OSF (namely a
private project). With this change a public project could be listed as
private when all that it means is unaccessible or unshared. This would be
correct for all components for both public and private, though I don't
recall if that was my intention initially. I believe Sara was in on that
discussion at one point, though she may not remember.

On Wed, Feb 24, 2016 at 4:40 PM, Steven Loria <notifications@github.com>
wrote:

> In website/templates/util/render_node.mako
> <https://github.com/CenterForOpenScience/osf.io/pull/4509#discussion_r54011173>
> :
>
> > @@ -143,13 +143,13 @@
> >          class="project list-group-item list-group-item-node">
> >          <p class="list-group-item-heading f-w-lg">
> >              %if summary['is_registration']:
> > -                Private Registration
> > +                Unshared Registration
>
> Why was this language changed? I don't see any logic that checks whether
> the user is viewing through a view-only link.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/4509/files#r54011173>
> .
>